CMakeLists.txt

```diff
@@ -76,7 +76,7 @@ add_definitions(-DOPENMP)
 #)
 
 find_package(CUDA)
-set (CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS}  --expt-extended-lambda  -gencode arch=compute_61,code=sm_61  -gencode arch=compute_52,code=sm_52 -gencode arch=compute_75,code=sm_75 -gencode arch=compute_86,code=sm_86 "  )
+#set (CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS}  --expt-extended-lambda  -gencode arch=compute_61,code=sm_61  -gencode arch=compute_52,code=sm_52 -gencode arch=compute_75,code=sm_75 -gencode arch=compute_86,code=sm_86 "  )
```

Comment:
Have you tried compiling with these cuda arch flags?

Reply:
Yes, this was causing a problem as I found the record. Might becuase of a merging issue. Successful on uncomment line 79